src/Widgets/TickListItemWidget/TickListItemWidgetClass.ts

```diff
@@ -0,0 +1,10 @@
+import { provideWidgetClass } from 'scrivito'
+
+export const TickListItemWidget = provideWidgetClass('TickListItemWidget', {
+  onlyInside: 'TickListWidget',
+  attributes: {
+    icon: 'string',
```

Comment:
The requirement came from a private discussion between Alex G. and me, so there is no document/chat that I can point to.

I also thought about setting only one icon for the whole list. I did not yet go this route, since we then would have to communicate the one icon from the container widget down to the child widget.

With this "one icon per row" approach you style icons differently. E.g. you could make a numbers list, e.g. with https://icons.getbootstrap.com/icons/1-circle/, https://icons.getbootstrap.com/icons/2-circle/ etc. Or you could pick icons matching to the content of the statement. In the example content I did not go that way, because it looked to playful for a serious business website.

Reply:
After some consideration I moved the setting up to the container.